cli/fathom_web/commands/train.py

```diff
@@ -54,12 +54,34 @@ def learn(learning_rate, iterations, x, y, confidence_threshold, num_prunes, val
     if stopped_early:
         print(f'Stopping early at iteration {t}, just before validation error rose.')
 
+    print(f'Successfully trained model using threshold {confidence_threshold}.')
```

Comment:
Do we need to say this? Is it actionable for the user? I've tried to minimize the output of `fathom train` (though I've failed for the "Building Fathomfox...done" bits).